Documentation/CrossPlatform/OculusQuestMRTK.md

```diff
@@ -32,8 +32,9 @@ This data provider does **NOT** use Unity's **XR Pipeline** or **Legacy XR Pipel
 1. Download and import [Oculus Integration](https://assetstore.unity.com/packages/tools/integration/oculus-integration-82022) from the Unity Asset Store. The latest version tested to
 work is 20.0.0. Older versions can be found from this [archive](https://developer.oculus.com/downloads/package/unity-integration-archive/)
 
-1. Navigate to Mixed Reality Toolkit > Utilities > Oculus > Integrate Oculus Integration Unity Modules. Doing this will update the asmdefs with definitions and references needed for the
-relevant Oculus Quest code to function. It will also update the csc file to filter out the obsolete warnings produced by the Oculus Integration assets. The MRTK repo contains a csc file that converts warnings to errors, this conversion halts the MRTK-Quest configuration process.
+1. Navigate to Mixed Reality Toolkit > Utilities > Oculus > Integrate Oculus Integration Unity Modules and run it. Doing this will update the asmdefs with definitions and references needed for the
+relevant Oculus Quest code to function. Additionally, it should populate the **Prefab reference** fields of the **OculusXRSDKDevicemanagerProfile** at */Assets/MRTK/Providers/Oculus/XRSDK/Profiles*.
```

Comment:
>at */Assets/MRTK/Providers/Oculus/XRSDK/Profiles*.

I'm not sure the best way to refer to this, but the profile won't be in the `Assets` folder if MRTK is imported via UPM

Reply:
We can just omit the file path then